Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -413,6 +409,9 @@ private void RegisterEncapsulateFieldRefactoringFactories(IWindsorContainer cont
             container.Register(Component.For<IEncapsulateFieldModelFactory>()
                 .ImplementedBy<EncapsulateFieldModelFactory>()
                 .LifestyleSingleton());
+            container.Kernel.Register(Component.For<IEncapsulateFieldReferenceReplacerFactory>()
```

Comment:
Why `.Kernel` here?

Reply:
No legitimate reason....`container.Kernel.Register` was a copy/paste transfer from how it was registered in `EncapsulateFieldTestsResolver`.  From some quick research, looks like the recommended way is just `container.Register` (as it's done in the IoCInstaller)...wish I had spotted this inconsistency earlier.   I'll get this changed and also update `EncapsulateFieldTestsResolver` to drop the use of `.Kernel` as well.

Reply:
> wish I had spotted this inconsistency earlier

*this is why we do reviews* 😉